rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import resolve from 'rollup-plugin-node-resolve';
+import { terser } from "rollup-plugin-terser";
+
+export default [
+  {
+    input: 'src/testTrack.js',
+    external: ['jquery', 'crypto'],
+    output: {
+      file: 'dist/testTrack.js',
+      format: 'esm'
+    },
+    plugins: [
+      resolve(),
+      commonjs(),
```

Comment:
does this need to have the `commonjs` plugin since it's the `esm` targetted file?

Reply:
the [plugin](https://github.com/rollup/rollup-plugin-commonjs) converts commonjs modules to es6 modules, so rollup can make use of them. its still needed here.